### DISCUSSION TITLE
Yattee is now available on the App Store! 🎉

### DISCUSSION THREAD

**Discussion by arekf** [yattee/yattee#331] [Announcements]
<a href="https://apps.apple.com/us/app/yattee/id1595136629?itsct=apps_box_badge&amp;itscg=30200" style="display: inline-block; overflow: hidden; border-radius: 13px; width: 250px; height: 83px;"><img src="https://tools.applemediaservices.com/api/badges/download-on-the-app-store/black/en-us?size=250x83&amp;releaseDate=1666742400&h=eda3695f90de92150bc4ec7505362a1a" alt="Download on the App Store" style="border-radius: 13px; width: 250px; height: 83px;"></a>

iOS and tvOS versions are now available to download from the App Store!

Big **Thank You** to everyone who participated in testing, bug reporting, translating and other activities that made this release possible.

**Comment by uthsobcb** [comment]
Search function is not working. Whatever i search it shows trending video list.

**Comment by arekf** [comment]
> Search function is not working. Whatever i search it shows trending video list.

You are using demo instance. You need to add your own in Locations settings.

**Reply by uthsobcb** [reply]
How can I add location?

**Reply by uthsobcb** [reply]
NVM, i figured it out. And working perfectly. Thanks

**Comment by AMD-NICK** [comment]
It's worth working on the user experience. I spent 30 minutes trying to figure out what's what)

But thank you very much for your work on this app! ❤️

**Comment by aniqueta** [comment]
Thank you! Should we expect the macOS app to be on the App Store soon too?

**Reply by arekf** [reply]
I tried to add it to Mac App Store several times but it keeps getting rejected on different reasons. I will keep trying sending them next updates and maybe we will get lucky one day. Or not 🙃
